### PR TITLE
Updates for Le Monde websites

### PR DESCRIPTION
**Pull request by techexo**
Addition of abonnes.lemonde.fr site config and suggestion of modification to strip annoying blocks for lemonde.fr.

### CODE REVIEW

**Comment by fivefilters** [comment]
Thanks! 👍

**Comment by techexo** [comment]
You're very welcome! (and sorry for pull requests 325 and 326, that I closed, I totally screwed them up ;-))

File: lemonde.fr.txt [hunk 1]
```diff
@@ -8,6 +8,8 @@ date: //time[@itemprop='dateModified']/@datetime
 # Publication date
 date: //time[@itemprop='datePublished']/@datetime
 
+# Remove "Lire aussi" blocks
+strip: //p[@class='lire']
```
Comment (j0k3r) [inline lemonde.fr.txt]: Hum, I found this information interesting most the time...

Comment (techexo) [inline lemonde.fr.txt]: Yes, it is clearly an aesthetic choice. However, I think that (in the context of Wallabag), what you want is the content of the article you see to read it later. If you want links to other things, you directly go see them and add them too to wallabag.

However, I won't be upset if this particular line is not kept; I will just continue to add it in my files :)

**Pull request merged**